packages/destination-actions/src/destinations/loops/sendEvent/index.ts

```diff
@@ -38,6 +38,13 @@ const action: ActionDefinition<Settings, Payload> = {
       default: {
         '@path': '$.userId'
       }
+    },
+    eventProperties: {
+      label: 'Event Properties',
+      description: 'Attributes maintained by your team.',
+      type: 'object',
+      required: false,
+      default: { '@path': '$.traits' }
```

Comment:
I think we want to use $.properties as the default here, traits are more properties of users specifically - https://segment.com/docs/connections/spec/track/